sandbox/rails-app/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM ruby:3.4.1
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
+RUN mkdir /myapp
+WORKDIR /myapp
+ADD Gemfile /myapp/Gemfile
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)